src/legacy/ui/public/chrome/api/base_path.test.ts

```diff
@@ -17,6 +17,7 @@
  * under the License.
  */
 
+import { basePathServiceMock } from '../../../../../core/public/base_path/base_path.mock';
```

Comment:
What do you think about exporting mocks at the root level of `core/public`? I imagine many plugins are going to be using these mocks and it'd best if they don't have to dig into the filesystem of core. I think adding a `core/public/mocks.ts` file that exports only the mocks that are needed (core start contracts) would be ideal.